app/experimenter/experiments/models/__init__.py

```diff
@@ -11,7 +10,6 @@
     ExperimentEmail,
     ExperimentManager,
     ExperimentVariant,
-    Locale,
     Preference,
     RolloutPreference,
     VariantPreferences,
```

Comment:
And the string references from `__all__` below ⬇️⬇️⬇️⬇️

Reply:
oooh looks like i missed that, will do